cheatsheets/gleam-for-elm-users.md

```diff
@@ -3,7 +3,18 @@ layout: page
 title: Gleam for Elm users
 ---
 
-- [Overview](#overview)
+## Overview
+
+Elm and Gleam have similar goals of providing a robust and sound type system with a friendly and approachable set of features.
+
+They have some differences in their output and focus. Where Elm compiles to JavaScript, Gleam initially aimed to compile to Erlang, and where Elm is best suited for front-end browser based applications, Gleam initially targets back-end and server-side application development.
```

Comment:
```suggestion
They have some differences in their output and focus. Where Elm compiles to JavaScript, Gleam compiles to Erlang, and where Elm is best suited for front-end browser based applications, Gleam initially targets back-end and server-side application development.
```
I thought this made it sound like it intended to but in the end did not compile to Erlang

Reply:
Yeah, good point. Thanks for changing it. I suspect it is a quirk of me not knowing how to think about the Javascript support that is landing. It might have crept into my language a bit as a result. Certainly better this way.